R/process-plate.R

```diff
@@ -60,21 +71,26 @@ is_valid_normalisation_type <- function(normalisation_type) {
 #' plate <- read_luminex_data(plate_file, layout_file)
 #'
 #' tmp_dir <- tempdir(check = TRUE)
-#' temporary_filepath <- file.path(tmp_dir, "output.csv")
-#' process_plate(plate, output_path = temporary_filepath)
 #' # create and save dataframe with computed dilutions
+#' process_plate(plate, output_dir = tmp_dir)
+#'
+#' # process plate without adjusting blanks and save the output to a file with a custom name
+#' process_plate(plate, filename = "plate_without_blanks_adjusted.csv",
+#'    output_dir = tmp_dir, adjust_blanks = FALSE)
+#'
 #'
 #' # nMFI normalisation
 #' process_plate(plate,
-#'   output_path = temporary_filepath,
+#'   output_dir = tmp_dir,
```

Comment:
while I personaly do not mind `tmp_dir` I would call it in the code that is exposed to users i.e. docs to `example_directory` or something along those lines

Reply:
what do you mean exactly? Do you want to print the `tmp_dir` path to the user? 
Or maybe you suggest renaming this variable to `example_directory`?

Reply:
I suggest renaming

Reply:
done